docs/user-guide/configure.md

```diff
@@ -179,6 +179,41 @@ For example:
 
 Reporters may use these severity levels to display problems or exit the process differently.
 
+Experimental feature: some rules support [message arguments](#message). For these rules, it is possible to use a function for `severity`, which would accept these arguments, allowing you to adjust the severity based on these arguments.
```

Comment:
suggestion

maybe a list would be better

Reply:
It seemed to me that it emphasises too much that a rule without `message` cannot have `severity`.
Anyway it's minor; Ill resolve.